platforms/org.eclipse.mita.platform.unittest/src/org/eclipse/mita/platform/unittest/UserStaticSizeInferrer.xtend

```diff
@@ -0,0 +1,36 @@
+package org.eclipse.mita.platform.unittest
+
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.mita.program.GeneratedFunctionDefinition
+import org.eclipse.mita.program.SignalInstance
+import org.eclipse.mita.program.inferrer.ElementSizeInferrer
+import org.eclipse.mita.program.inferrer.StaticValueInferrer
+import org.eclipse.mita.program.model.ModelUtils
+import org.yakindu.base.expressions.expressions.FeatureCall
+
+class UserStaticSizeInferrer extends ElementSizeInferrer {
```

Comment:
The "User" is in there because the user specifies what the maximum length is. Maybe that is important for naming, but if you like "PlatformResourceSizeInferrer" better we can use that as well.

Reply:
How about `GenericPlatformSizeInferrer`?